bench/workgen/runner/workgen_perf_check.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
```

Comment:
Who calls this script? Can you please put a comment here stating the usage of this script?

Reply:
This script will be used in an evergreen task, see WT-7679.
I will add a comment about it.